packages/office-ui-fabric-react/src/components/TeachingBubble/docs/new/TeachingBubbleBestPractices.md

```diff
@@ -0,0 +1,15 @@
+### Layout
+
+- TeachingBubbles can be used in sequence to walk people through complex, multistep interactions only. And only show one TeachingBubble at a time.
+  - A maximum of no more than 3 sequenced TeachingBubbles should be used in a single experience.
+  - Sequenced TeachingBubbles should have “x of y” text to indicate progress through the sequence. For example, the first TeachingBubble in a sequence might be “1 of 3”.)
+  - The action button on a sequenced TeachingBubble should read “Next” and the final sequenced TeachingBubble should have an action button with text “Got it”, that closes the experience.
+- Always place the primary button on the left, the secondary button just to the right of it.
+- Show only one primary button that inherits theme color at rest state. In the event there are more than two buttons with equal priority, all buttons should have neutral backgrounds.
+
+### Content
+
+- **Title**: You have 2550 characters (including spaces) to draw people in and get them interested. Limit to one line ofd text, aAnd use sentence casing (capitalize only the first word and any proper nouns) with no punctuation.
+- **Body text**: Lead with why the feature is useful rather than describe what it is. What does it make possible for our customers? Keep it within 120512 characters (including spaces).
```

Comment:
```suggestion
- **Body text**: Lead with why the feature is useful rather than describing what it is. What does it make possible for our customers? Keep it within 120512 characters (including spaces).
```